Training/Api/Data/NoteInterface.php

```diff
@@ -73,14 +73,14 @@ public function setNoteId(int $noteId);
      *
      * @return string
      */
-    public function getNoteText();
```

Comment:
What the purpose of this change? Moreover, reed about service contract https://devdocs.magento.com/guides/v2.4/extension-dev-guide/service-contracts/service-contracts.html#msc-benefits
And even you decided to rename method, you also have to rename method in the class that implemented that interface